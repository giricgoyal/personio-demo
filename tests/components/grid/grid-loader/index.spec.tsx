import React from 'react'
import GridLoader from 'src/components/grid/grid-loader'
import { render } from 'tests/test-utils'

describe('components/grid/grid-loader', () => {
    let container, rerender

    beforeEach(() => {
        ;({ container, rerender } = render(<GridLoader />))
    })

    test('should not render and match snapshot', () => {
        expect(container).toMatchSnapshot()
    })

    describe('When lodaing', () => {
        beforeEach(() => {
            rerender(<GridLoader isLoading />)
        })

        test('should render and match snapshot', () => {
            expect(container).toMatchSnapshot()
        })
    })
})
