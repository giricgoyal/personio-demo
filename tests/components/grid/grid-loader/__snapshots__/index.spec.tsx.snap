// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/grid/grid-loader When lodaing should render and match snapshot 1`] = `
<div>
  <div
    class="grid__loading"
  >
    <i
      class="fas fa-spinner fa-2x fa-spin"
    />
  </div>
</div>
`;

exports[`components/grid/grid-loader should not render and match snapshot 1`] = `<div />`;
